src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -92,22 +92,26 @@ export default (accessToken, userID, opts) => {
       { headers: true }
     ),
     artworkLoader: gravityLoader((id) => `artwork/${id}`),
+    // DEPRECATED: Use artworkImportV2CreateArtistAssignmentsLoader instead
```

Comment:
These flags will be removed once the Volt side is updated, and are used as context for FE work.